### PR TITLE
h2olog: add tid to identify QUIC connections (QUIC conn master_id + tid)

### PR DESCRIPTION
**Pull request by gfx**
Alternative to #2719. Adding a common field `tid` for h2o's tid.

### CODE REVIEW

File: src/h2olog/misc/gen_raw_tracer.py [hunk 1]
```diff
@@ -374,7 +374,7 @@ def build_tracer(context, metadata):
   c = r"""// %s
 int %s(struct pt_regs *ctx) {
   const void *buf = NULL;
-  struct h2olog_event_t event = { .id = %s };
+  struct h2olog_event_t event = { .id = %s, .tid = get_h2o_tid(), };
```
Comment (kazuho) [inline src/h2olog/misc/gen_raw_tracer.py:377]: How about doing `(uint32_t)bpf_get_current_pid_tgid()` here directly?

There's no need to pay the costs of calling another eBPF function, the costs being overhead in terms of execution time, code size, potential interoperability issues (IIRC, you cannot call another eBPF function in older kernels).

**Review by kazuho — COMMENTED** [review]
Thank you for the PR. This looks good modulo the point below.

File: src/h2olog/misc/gen_raw_tracer.py (hunk 1 above)
Comment (gfx) [reply src/h2olog/misc/gen_raw_tracer.py]: Adding `inline` is also ok, but in fact there's no need to define an extra utility.  Fixed 22762f1f1

**Comment by kazuho** [comment]
Thank you for the changes. Merged.

**Pull request merged**